### ISSUE TITLE
Support organization repositoried in github integration

### ISSUE THREAD

**Issue by crimson-med** [thepracticaldev/dev.to#8971]
**Is your feature request related to a problem? Please describe.**

I just tried updating my pinned repositories through the `integrations` menu and saw that repos I moved to my organization are not select-able.

**Describe the solution you'd like**

Being able to choose any repo owned by me even in an organization

**Additional context**

Wanted to add my new package but it wasn't appearing:
https://github.com/argus-inc/fluct

**Comment by citizen428** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
Hi @crimson-med, thanks for the request, I think this is a duplicate of https://github.com/thepracticaldev/dev.to/issues/243

Please feel free to add a comment there as well. 

Thank you!
